DuckDuckGo/TabViewController.swift

```diff
@@ -174,29 +178,43 @@ class TabViewController: WebViewController {
     }
 
     func launchBrowsingMenu() {
+        Pixel.fire(pixel: .browsingMenuOpened)
+
         guard let button = chromeDelegate?.omniBar.menuButton else { return }
         let alert = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
         alert.addAction(refreshAction())
         alert.addAction(newTabAction())
 
-        if let link = link {
+        if let link = link, !isError {
+            alert.addAction(saveBookmarkAction(forLink: link))
+            alert.addAction(toggleDesktopSiteAction(forUrl: link.url))
```

Comment:
Does it make sense to list this after the share option for consistency with the Android app?

Reply:
This is the order agreed with Chris - I'll take that question back to Asana.

Reply:
Making the change as you suggest - will let Chris object